jumpscale/packages/vdc_dashboard/sals/solutions_chatflow.py

```diff
@@ -503,18 +503,24 @@ def initializing(self, timeout=300):
                 Reason: {{reason}}
                 """
         start_time = time()
-        while time() - start_time <= POD_INITIALIZING_TIMEOUT:
+        while time() - start_time <= pod_initalizing_timeout:
             if self.chart_pods_started():
                 break
             gevent.sleep(1)
 
-        if not self.chart_pods_started() and self.chart_resource_failure():
+        if self.chart_resource_failure():
             stop_message = error_message_template.format(
                 reason="Couldn't find resources in the cluster for the solution"
             )
             self.k8s_client.execute_native_cmd(f"kubectl delete ns {self.chart_name}-{self.config.release_name}")
             self.stop(dedent(stop_message))
 
+
+        if not self.chart_pods_started():
+            stop_message = error_message_template.format(reason="Pods initialization timed out")
+            self.k8s_client.execute_native_cmd(f"kubectl delete ns {self.chart_name}-{self.release_name}")
+            self.stop(dedent(stop_message))
```

Comment:
we can add remove cleaning subdomains too and that fulfills https://github.com/threefoldtech/js-sdk/issues/2914

Reply:
sure I will work on this in another PR request as it is a separate issue and happening in another places in the code  
https://github.com/threefoldtech/js-sdk/issues/2914